src/websocket/socket.io.ts

```diff
@@ -44,6 +44,7 @@ export const createServer = (env: RouteInitEnvironment) => {
         pingInterval: 5000,     // socket.io 2.x default is 25000
         pingTimeout: 5000,      // socket.io 2.x default is 5000, 3.x default = 20000
         maxHttpBufferSize: maxPayloadBytes,
+        path: env.root + "ws",
```

Comment:
Keep this at `'socket.io'` instead of `'ws'` for backward compatibility & possible future move to other websocket library